upstream-community-operators/federation/federation.v0.0.7.clusterserviceversion.yaml

```diff
@@ -3,14 +3,14 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: federation.v0.0.6
+  name: federation.v0.0.7
   namespace: placeholder
   annotations:
     capabilities: Basic Install
     categories: "OpenShift Optional, Integration & Delivery"
     description: Gain Hybrid Cloud capabilities between your clusters with Kubernetes Federation.
     certified: "false"
-    containerImage: "quay.io/openshift/origin-federation-controller:v4.0.0"
+    containerImage: "quay.io/kubernetes-multicluster/federation-v2:v0.0.7"
     createdAt: "2019-01-01T00:00:00Z"
 spec:
```

Comment:
Please add a `spec.maintainers` section. Please refer to the docs for what this should look like: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#example-csv

Reply:
I really do not have a suitable email address for this field right now. My team needs to discuss what the right value is for this field to take on. @robszumski can I get an exception on this one for now?

Reply:
yes, that is ok. in the future, a user list or something would be great if it exists (I understand it doesn't now)